mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule ExJsonSchema.Mixfile do
 
   defp deps do
     [
+      {:decimal, "~> 2.0"},
```

Comment:
Unfortunately, this means that the "zero dependencies" claim in the README has to be removed as well. But since there's no way to correctly implement `multipleOf` with Floats in Elixir, I'm fine with that 🤷‍♂️